internal/cache/file/cache_handler.go

```diff
@@ -70,8 +70,8 @@ func (chr *CacheHandler) createLocalFileReadHandle(objectName string, bucketName
 }
 
 // cleanUpEvictedFile is a utility method called for the evicted/deleted fileInfo.
-// As part of execution, it stops and removes the download job, and deletes the cache
-// file.
+// As part of execution, it stops and truncates and removes the download job,
```

Comment:
nit: As part of execution, it (a) stops and removes the download job, (b) truncate and deletes the cache file.